cocoa/tests_backend/widgets/activityindicator.py

```diff
@@ -5,3 +5,7 @@
 
 class ActivityIndicatorProbe(SimpleProbe):
     native_class = NSProgressIndicator
+
+    @property
+    def is_hidden(self):
```

Comment:
It would be worth being explicit that we're talking about the *spinner* visibility here. Depending on the implementation, it would be possible to have a widget that is "visible" in the sense of Toga's drawing instructions, but that that displays no content because the *spinner* is hidden. iOS got caught on this because spinner visibility and widget visibility are treated as the same thing; but that's no *necessarily* the same.

There's already a prove for evaluating visibility

Reply:
Hey @freakboy3742 ! I will have a look deeper into this - it looks like it is possible to do in iOS and macOS but I still need to figure out for the Linux and Windows implementations how it would be done

Reply:
Winforms doesn't have an ActivityIndicator, so the solution there will be easy :-)

As for Linux - if we can verify that the GTK widget behaves as expected (i.e., hidden and running are different properties, and a stopped indicator is defined as not visible, but not hidden), then it might be sufficient to just check the `active` property.